src/test/java/org/gridsuite/ds/server/service/client/AbstractRestClientTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2022-2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.ds.server.service.client;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.HttpUrl;
+import okhttp3.mockwebserver.Dispatcher;
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.http.codec.json.Jackson2JsonDecoder;
+import org.springframework.http.codec.json.Jackson2JsonEncoder;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.reactive.function.client.ExchangeStrategies;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.io.IOException;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+@RunWith(SpringRunner.class)
+public abstract class AbstractRestClientTest {
+
+    protected WebClient.Builder webClientBuilder;
+
+    protected MockWebServer server;
+
+    protected final ObjectMapper objectMapper = createObjectMapper();
+
+    public final Logger getLogger() {
```

Comment:
Why are you not creating a static "protected final Logger LOGGER" ?

Reply:
I think this is only a choice! Since i do a few log so I use a method to get logger on-demand.

Anyway, I have corrected as yours suggestion but without static.